cmd/generator/main.go

```diff
@@ -120,28 +105,21 @@ var generatorCmd = &cobra.Command{
 		for _, repositoryName := range repositoryNames {
 			logger.Debugf("querying repository %s", repositoryName)
 
-			releasePlugins, err := getReleasePlugins(ctx, client, repositoryName, includePreRelease, existingPlugins)
+			var releasePlugins []*model.Plugin
+			releasePlugins, err = getReleasePlugins(ctx, client, repositoryName, includePreRelease, existingPlugins)
 			if err != nil {
 				return errors.Wrapf(err, "failed to release plugin for repository %s", repositoryName)
 			}
 
 			for _, plugin := range releasePlugins {
 				if len(plugin.IconData) == 0 {
 					if iconPath, ok := iconPaths[repositoryName]; ok {
-						icon, err := getIcon(ctx, iconPath)
+						var iconData string
+						iconData, err = getIconDataFromPath(iconPath)
 						if err != nil {
 							return errors.Wrapf(err, "failed to fetch icon for repository %s", repositoryName)
 						}
```

Comment:
nit: inline if

Reply:
At the end it's a style question and I like the style this way :wink: